lua/telescope/utils.lua

```diff
@@ -170,6 +170,113 @@ utils.filter_symbols = function(results, opts, post_filter)
   end
 end
 
+utils.is_path_hidden = function(opts, path_display)
```

Comment:
stop moving functions around if you dont have to, now its hard to get an understanding if something was changed

Reply:
That was my intention, but I noticed I moved them to the wrong place. Pushed a new commit to resolve this.